setup.md

```diff
@@ -0,0 +1,130 @@
+iOS or Android.
+
+[System requirements](#system_requirements)<br>
+[iOS setup](#ios_setup)<br>
+[Android setup](#android_setup)<br>
+[Get the Flutter SDK](#get_the_flutter_sdk)<br>
+[Atom editor](#atom_editor)
+
+## System requirements ##
+
+To install and run Flutter, your development environment must meet these minimum requirements.
+
+* **Operating Systems**: Mac or Linux (64-bit). Windows support is planned.
+* **Tools**: Flutter depends on these command-line tools being available on your environment.
+  * `bash`, `mkdir`, `rm`, `git`, `curl`, `unzip`
+
+## iOS setup ##
+
+To develop Flutter apps for iOS, you need a Mac with Xcode 7.2 or newer:
+
+1. Install Xcode 7.2 or newer (via [web download](https://developer.apple.com/xcode/) or
+the [Mac App Store](https://itunes.apple.com/us/app/xcode/id497799835)).
+2.  Make sure the Xcode EULA is signed by either opening Xcode once and confirming or
+running `sudo xcodebuild -license` from the command-line.
+
+With Xcode, you’ll be able to run Flutter apps on the iOS simulator.
+
+### Set up the iOS simulator ###
+
+To prepare to run and test your Flutter app on the iOS simulator, follow these steps:
+
+1. On your Mac, find the Simulator via Spotlight or by using the following command:
+
+    ```
+    $ open -a Simulator
+    ```
+
+2. Make sure your simulator is using a 64-bit device (iPhone 5s or later) by checking the settings
+in the simulator's **Hardware > Device** menu.
+3. Depending on your development machine's screen size, simulated high-screen-density iOS devices
+may overflow your screen. Set the device scale under the **Window > Scale** menu in the simulator.
+
+
+### Deploy to iOS devices ###
+
+To deploy your Flutter app to a physical iOS device, you’ll need some additional tools:
+
+1. Install [homebrew](http://brew.sh/).
+2. Open the terminal and run these command to install the toools for deploying Flutter apps to
+iOS devices.
+
+    <pre>
+    $ brew tap flutter/flutter
+    $ brew install ideviceinstaller ios-deploy
+    </pre>
+
+## Android setup ##
+
+To develop Flutter apps for Android, you can use either a Mac or a Linux (64-bit) machine:
+
+1. Install [Android Studio](https//developer.android.com/sdk/index.html).
+2. From Android Studio, install the latest Android SDK and Android SDK Platform-Tools, as described
+[here](https://developer.android.com/sdk/installing/adding-packages.html).
+
+### Set up your Android device
+
+To prepare to run and test your Flutter app on Android, you'll need an Android device running
+Android 4.1 (API level 16) or higher.
+
+1. Enable developer mode on your device by visiting **`Settings > About phone`** and
+   tapping the **`Build number`** property seven times.
+2. Enable **`Android debugging`** in **`Settings > Developer options`**.
+3. Using a USB cable, plug your phone into your computer. If prompted on your
+   device, authorize your computer to access your device.
```

Comment:
on this line you indent the list item text and on some other lines you don't; either is fine by me but we should be consistent throughout the file